### ISSUE TITLE
1.1.0 destroys globals after the first test method in PHPUnit

### ISSUE THREAD

**Issue by mrclay** [sebastianbergmann/global-state#5]
Our bootstrap sets a couple global `stdClass` vars and we use the default options in [our config](https://github.com/Elgg/Elgg/blob/2.0/phpunit.xml). After a recent composer update I noticed these globals get removed after the first test method in the suite, leading to `Creating default object from empty value` warnings when we write to them.

Forcing composer to use the "1.0" global-state fixes this problem.

**Comment by sebastianbergmann** [comment]
Can you please provide a minimal, self-contained test case that demonstrates this? Thanks!

**Comment by mrclay** [comment]
Sure: https://github.com/mrclay/globals_state_5
```
git clone git@github.com:mrclay/globals_state_5.git
cd globals_state_5
composer install
./vendor/phpunit/phpunit/phpunit
```

**Comment by mrclay** [comment]
I pushed a more straightforward test of `isset($GLOBALS['varname'])`

**Comment by mrclay** [comment]
`Snapshot::canBeSerialized()` returns false for a `stdClass`.

**Comment by sebastianbergmann** [comment]
Interesting. IMHO that's a bug in PHP. I'll implement a workaround for this, of course.

**Comment by mrclay** [comment]
Whitelist `stdClass` as serializable?

**Comment by mrclay** [comment]
Fix #6
